autoapi/mappers/dotnet.py

```diff
@@ -355,6 +355,14 @@ def ref_short_name(self):
         '''Same as above, return the truncated name instead'''
         return self.ref_name.split('.')[-1]
 
+    @property
+    def include_path(self):
+        """Return 'absolute' path without regarding OS path separator"""
+        parts = ['/autoapi']
```

Comment:
This should probably come from the setting that sets the path of the output, which won't necessarily always be `autoapi`